bridge-www/src/App.tsx

```diff
@@ -53,6 +54,7 @@ function App() {
   }
 
   console.log(contract, contractAddress, accounts, account, amount)
+  const amountInEthereum = new Decimal(amount || "0").mul(new Decimal(10).pow(18));
```

Comment:
I guess we can use `useEffect` and `useState` here too. and IMO that is a more reactive way.

```suggestion
  useEffect(() => {
    setAmountInEth(new Decimal(amount || "0").mul(new Decimal(10).pow(18)));
  }, [amount]);
```

Reply:
Thanks to good suggestion. But I found [`useMemo`][use-memo] hook for memoization with dependencies. It looks better to me because I don't think `new Decimal(amount || "0").mul(new Decimal(10).pow(18))` will occur side-effect 🤔 .

As code:

```suggestion
  const amountInEthereum = useMemo<Decimal>(() => new Decimal(amount || "0").mul(new Decimal(10).pow(18)), [amount]);
```

How do you think about it?

[use-memo]: https://reactjs.org/docs/hooks-reference.html#usememo

Reply:
`useMemo` is way better. 👍